### ISSUE TITLE
Accessibility:Input Group: The “dollar” info ($) and value “0.00” within input groups is not conveyed when input groups are accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3492]
Description: The “dollar” info ($) and value “0.00” within input groups is not conveyed when input groups are accessed. 

Expected: information about each & every UI should be read out correctly to the user. 

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94650260-f4210a00-0313-11eb-9d0c-a245f383aa49.png)

![image](https://user-images.githubusercontent.com/32538291/94650265-f7b49100-0313-11eb-9687-7f7cbe7601ca.png)

![image](https://user-images.githubusercontent.com/32538291/94650269-faaf8180-0313-11eb-8047-fbba8d949505.png)

![image](https://user-images.githubusercontent.com/32538291/94650280-fdaa7200-0313-11eb-93b4-e055bf6596ad.png)

![image](https://user-images.githubusercontent.com/32538291/94650284-00a56280-0314-11eb-8397-2ea9c19f15f8.png)

**Comment by puru-hk** [comment]
@Sanjay-1312 could you please retest this issue.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue is fixed, hence closing the Ticket.

Thanks and Regards
Sanjay
